src/main/java/jewellery/inventory/JewelleryInventoryApplication.java

```diff
@@ -8,6 +8,6 @@ public class JewelleryInventoryApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(JewelleryInventoryApplication.class, args);
-	}
 
+	}
```

Comment:
Can you try to reset this file to the state it was? We don't want to have obsolete changes in the commits. You can try to do `git checkout origin/main src/main/java/jewellery/inventory/JewelleryInventoryApplication.java
` this will reset just the JewelleryInventoryApplication file to the state it was in main.